server/views/pages/goal/add-note/index.njk

```diff
@@ -1,14 +1,13 @@
 {% extends "../../../partials/layout.njk" %}
-{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 
 {% set pageId = "add-note" %}
-{% set pageTitle = applicationName + " - Home" %}
-{% set mainClasses = "app-container govuk-body" %}
+{% set pageTitle = "Add a note to this goal (optional)" %}
 
 {% block beforeContent %}
-  {% include "../../../partials/breadCrumb.njk" %}
+  {{ govukBackLink({
```

Comment:
100% we need to get rid of the back links

A discussion before you started agreed that *all* backlinks were out of scope. Its easier to explain why over a chat; and I'd be keen to hear your thoughts and view on it, especially our concerns - but for here and now the back links are 100% out of scope.

Please can you remove them from all the pages you added them to? (sorry!)

Reply:
But happy to remove them if they are out of scope of this change and wait for a JIRA ticket.